### DISCUSSION TITLE
so-elastic-auth not seen

### DISCUSSION THREAD

**Discussion by Cosmin98-sys** [Security-Onion-Solutions/securityonion#1718] [Unsupported Versions] [answered]
Hi!
I've enabled Elastic Features and now I'm trying to make Elastic Auth, but "so-elastic-auth" command is not existent.
What can I do ?

Thank you !

**Comment by dougburks** [comment, marked as answer]
`so-elastic-auth` is currently only available in Security Onion 16.04. 

We do not yet support native Elastic authentication in Security Onion 2.

**Reply by Cosmin98-sys** [reply]
And there is not another way to create a superuser in Security Onion 2?
Or in how much time will be available ?

Thanks!

**Reply by dougburks** [reply]
If you have not enabled native Elastic authentication, then there is no need to create a superuser as all user accounts have access to Elastic.

**Reply by Cosmin98-sys** [reply]
All that I've done after installing Security Onion 2 was to use the command so-elastic-features and after that I wanted to make some detection rules in Kibana and I cannot. And if I want to create some agent for Security part I still cannot because I need superuser account.
So I don't know if I've enabled it.

**Reply by Cosmin98-sys** [reply]
@dougburks how can I disable that native Elastic auth ?

Thank you !

**Reply by dougburks** [reply]
As I mentioned above, Security Onion 2 does not support native Elastic auth at this time.
